packages/atmosphere/src/providers/instagram/processor.ts

```
@@ -3,8 +3,33 @@ import type {
   APISubstatus,
   APIUser,
   APIVideo,
+  APIVideoFormat,
   APIPhoto
 } from '../../types/api-schemas.js';
+import {
+  parseDashBandwidthByHeight,
+  parseDashPresentationDurationSec
+} from './extractors.js';
```

ESLint

[error] Replace `⏎··parseDashBandwidthByHeight,⏎··parseDashPresentationDurationSec⏎` with `·parseDashBandwidthByHeight,·parseDashPresentationDurationSec·`

(prettier/prettier)